package.json

```diff
@@ -15,7 +15,7 @@
     "@typespec/http-specs": "0.1.0-alpha.22",
     "@typespec/spector": "0.1.0-alpha.14",
     "@azure-tools/azure-http-specs": "0.1.0-alpha.17",
-    "@azure-tools/typespec-azure-resource-manager": "0.56.0",
+    "@azure-tools/typespec-azure-resource-manager": "0.56.1",
```

Comment:
Keep this as 0.56.0. Do `npm install` again.

Reply:
I'm wrong. Revert this commit first. https://github.com/Azure/autorest.csharp/commit/692bbfbd4840804d10daea5c354115a0129365db